.kokoro/build.sh

```diff
@@ -41,5 +41,6 @@ python3.6 -m nox --version
 if [[ -n "${NOX_SESSION:-}" ]]; then
     python3.6 -m nox -s "${NOX_SESSION:-}"
 else
-    python3.6 -m nox
+    # docfx session runs in the 'docs-presubmit' presubmit
+    python3.6 -m nox -k "not docfx"
```

Comment:
I'm curious if we should be modifying the `noxfile` instead? I recall there's a way to mark certain sessions as "default"

Reply:
Looks like [there is](https://nox.thea.codes/en/stable/config.html?highlight=default#modifying-nox-s-behavior-in-the-noxfile)! I modified the noxfile.py. PTAL